keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -551,16 +575,32 @@ Below are some examples to consider, in addition to the aforementioned [maturity
 -->
 #### Alpha
 
-- Feature implemented behind a feature flag
+- Feature implemented behind a feature gate
 - Integration tests added for all relevant verbs (POST, PUT, and PATCH)
 
-<!--
 #### Beta
 
-- Gather feedback from developers and surveys
-- Complete features A, B, C
-- Additional tests are in Testgrid and linked in KEP
+- kubectl validate flag offers ability to perform server-side validation
+- All follow-ups from alpha are complete (see below)
+
+##### Alpha Follow Ups
+
+These are some tasks that were initially planned for alpha, but did not make it
+and should be addressed before beta is considered complete.
```

Comment:
drop the lines/header describing the alpha → beta shift and just include them as beta criteria

Reply:
done